NanoProd/config/skim_htt.yaml

```diff
@@ -98,6 +111,57 @@ selection: "
   int n_fatJet = FatJet_pt[fatJet_sel].size();
   if(n_fatJet >= 1 || nFatJet >= 2) return true;
 
+  auto boostedtau_base_sel = boostedTau_pt > 18 && abs(boostedTau_eta) < 2.5;
+  auto boostedtau_deepTau_sel = boostedTau_rawBoostedDeepTauRunIIv2p0VSjet > 0.72 && boostedTau_rawBoostedDeepTauRunIIv2p0VSe > 0.1 && boostedTau_rawBoostedDeepTauRunIIv2p0VSmu > 0.003;
+  auto boostedtau_sel = boostedtau_base_sel && boostedtau_deepTau_sel;
+  int n_boostedtaus = boostedTau_pt[boostedtau_sel].size();
+  int n_boosted_pairs = 0;
+  float maxR2 = 0.8*0.8;
+  float minR2 = 0.1*0.1;
```

Comment:
0.1 is about size of tau cone so it prevents that one boosted tau does not share constituents with other one or e or mu. In the case of two taus it should be true by construction, but can be different for tau and lepton. 0.01 can be fine for 1p taus but not for 3p. However, the only issue with too small minR is that some events can be accepted due to double counting, so if it is not the issue 0.01 will work too.

Reply:
Boosted taus are reconstructed from PF candidates that belong to the given sub-jet of a FatJet, correct? If yes, if dR between two sub jets is less than 0.1 muon could belong to the different sub-jet and will be “invisible” during the tau reconstruction. It might be possible to separate these cases from the cases when moun belong to the same sub-jet (and so interfering with tau reconstruction) at the analysis level, if needed. That was my reasoning to use 0.01 at the skim level.

Reply:
What you wrote is true, so I agree that  0.01 is enough to be a safeguard. Will you implement it yourself, e.g. as a suggestion? I have to run now and will be able do it not earlier that tomorrow afternoon (while accepting suggestion can be faster). Sorry for it.